app/controllers/submissions_controller.rb

```diff
@@ -41,7 +41,13 @@ def tag
   end
 
   def goto
```

Comment:
I don't see why Rubocop is mad about this - not sure where it's seeing branches..

To disable it, use:
```suggestion
  def goto # rubocop:disable Metrics/AbcSize
```

Reply:
Thank you! :)